content/core-features/channels.textile

```diff
@@ -0,0 +1,31 @@
+---
+title: Channels
+section: core-features
+index: 23
+---
+
+Ably aggregates all it's data into named units of distribution, referred to as "channels". Channels offer a way to implement the Publish-Subscribe (Pub/Sub) architectural pattern, which is a popular pattern used for realtime data delivery. 
+
+The Publish-Subscribe messaging pattern lets any number of publishers publish data to a channel, which could be subscribed to by any number of subscribers. The key thing to note about Pub/Sub is that publishers and subscribers are completely decoupled as explained further. Besides, once subscribed, the subscribers no longer have to poll the server or data provider to check if there is any new data that they need to be aware of; instead, they will be notified of it as it becomes available.
+
+h2(#understanding-pubsub). Understanding Pub/Sub with an example
+
+To understand Publish-Subscribe in more detail, let's consider an example of location tracking of a vehicle in realtime. In this case, the vehicle whose location is to be tracked acts as a publisher, while the user intending to receive the location updates acts as a subscriber. This scenario is illustrated below.
+
+<a href="/images/diagrams/pub-sub.png" target="_blank">
+  <img src="/images/diagrams/pub-sub.png" style="width: 100%" alt="Pub/Sub in Ably">
+</a>
+
+As you can see, in order to accomplish this scenario, the client would subscribe to the location channel, to receive updates continuously being published by the vehicle to the same channel. Since the location tracking needs to be live, one would use the realtime library to implement this. Ably's Realtime library uses WebSockets transport protocol under the hood; thus the connection remains open for the whole duration that the app is running for.
+
+h2(#understanding-decoupled-clients). Understanding de-coupling of Pub/Sub clients
+
+Ably's Data Stream Network supports the Publish-Subscribe messaging pattern via the concept of channels as explained in the previous sections. The various clients of Ably's Data Stream Network can be quite diverse, as shown in the illustration below. This means that while a publisher might be a sensor working with MQTT, the subscriber could be a web browser working with JavaScript.
```

Comment:
It'd be good here to clearly state what we mean, rather than only providing the example. `With Ably, you are able to use any number of devices and languages with one another. A publisher might be a sensor...`.